src/common/src/main/scala/osmesa/common/model/Change.scala

```diff
@@ -0,0 +1,114 @@
+package osmesa.common.model
+
+import java.sql.Timestamp
+
+import org.apache.spark.sql.types._
+import org.joda.time.DateTime
+import osmesa.common.model.Actions.Action
+
+import scala.xml.Node
+
+// TODO at some point user metadata (changeset, uid, user, timestamp?) should become options, as they may not be
+// available
+case class Change(id: Long,
+                  `type`: String,
+                  tags: Map[String, String],
+                  lat: Option[Double],
+                  lon: Option[Double],
```

Comment:
`java.math.BigDecimal`s are preferable here so that output can be unioned directly with the output of `osm2orc`. However, serialization in https://github.com/azavea/osmesa/pull/106/files#diff-7244aaf00b91b3c9d509c52c463e0388R48 (`ReplicationStreamBatchReader.get()`) results in `null` values no matter what (they're valid when passing into the encoder, then 💥, they're `null`.

Perhaps it's time to make a breaking change to `osm2orc` and change the `BigDecimal`s to `double`s (while also removing `nds.ref` (`Seq[Row]` where `Row` is a single column containing a struct named `ref`) in favor of making it `Seq[Long]`).

Reply:
Handled in #115. `Double`s and `BigDecimal`s should be handled interchangeably within OSMesa.